pages/About/index.jsx

```diff
@@ -0,0 +1,33 @@
+import Footer from "../../Components/layouts/Footer";
+import Header from "../../Components/layouts/Header";
+
+export default function About(){
+  return (
+    <>
```

Comment:
you don't need fragement here i think

Reply:
instead you can create a page layout that will hold a the header and footer , and children as props